### ISSUE TITLE
Nervos DAO will not sync past 4,595,196

### ISSUE THREAD

**Issue by smonistere** [nervosnetwork/neuron#2053]
**Describe the bug**
I seem to only be able to sync to, 4,595,196.I can see staked amount and can start the withdraw process etc. I have cleared the cache twice. I have deleted the wallet, reinstalled the program etc. It shows available amount but also says its updating.

**Environment (please complete the following information):**
 - OS & Version: Windows 10 Home 20H2
 - Neuron Version 0.35.0
 - Block Number: 4,595,196

**Steps To Reproduce**
 Just open Neuron and it will initially say 100% synced for a second 4,595,196/4,595,196 but then it will start going backwards to about 97.13% 4,595,196/4,731,196

**Comment by smonistere** [comment]
If I am not fully synced does that also mean that I can not withdraw from the staking? My listed compensated period is not right at all. I just want to withdraw it all and place it on the portal wallet and stake that way.a

**Comment by damienkeating** [comment]
similar issue here. The Neuron wallet stopped syncing properly recently. After clearing the cache, the wallet begins to sync, but after a while it stops at different blocks, sometimes block 970, sometimes block 1475003. It could not be fully synced.

**Comment by smonistere** [comment]
I could never get it fixed on that laptop. I downloaded it onto another laptop and it worked fine. I was able to pull my funds out and just move over to Portal Wallet much easier to stake. 
    On Thursday, August 19, 2021, 06:41:16 PM CDT, Damien Keating ***@***.***> wrote:  
 
 


similar issue here. The Neuron wallet stopped syncing properly recently. After clearing the cache, the wallet begins to sync, but after a while it stops at different blocks, sometimes block 970, sometimes block 1475003. It could not be fully synced.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
